src/backend/storeManagers/hyperplay/games.ts

```diff
@@ -934,6 +944,15 @@ export async function install(
       channelName
     })
 
+    const isMod = gameInfo.type === 'mod'
+    if (isMod) {
```

Comment:
this should be if game is market wars for now since it is not general to all mods